index.html

```diff
@@ -87,106 +87,11 @@ <h3>Multi-Post Stories</h3>
                                 </ul>
                             </div>
                             <button type="button" class="workButton" tabindex="10">See Project</button>
-                        </div>
+                        </div>-->
```

Comment:
- [ ] I kindly suggest that you please remove commented-out code from your work as it doesn't make your code neat.